openvm/src/lib.rs

```diff
@@ -1616,16 +1616,16 @@ mod tests {
         let powdr_metrics_sum = AirMetrics {
             widths: AirWidths {
                 preprocessed: 0,
-                main: 4824,
+                main: 4831,
```

Comment:
This test gets worse somehow, but it seems hard to debug. But I think it could just be because now it adds more precompiles somehow?

Reply:
Yes, if we save columns in each, it may well be leaving space for more apcs, leading to more columns